avocado/utils/linux_modules.py

```diff
@@ -96,7 +101,22 @@ def loaded_module_info(module_name):
     :rtype: dict
     """
     l_raw = process.system_output('/sbin/lsmod')
-    return parse_lsmod_for_module(l_raw, module_name)
+    modinfo_dic = parse_lsmod_for_module(l_raw, module_name)
+    output = process.system_output("/sbin/modinfo %s" % module_name)
+    if output:
+        param_list = []
+        for line in output.splitlines():
```

Comment:
Sure, will implement `if/elifs` as mentioned.
As for `params` values, getting values per `pci_address` makes more sense. That was the reason I did not implement it here. What do you suggest @ldoktor